helm/draft-annotations-api/templates/service.yaml

```diff
@@ -3,16 +3,17 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: {{.Values.service.name}} 
+  name: {{.Values.service.name}}
   labels:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     app: {{.Values.service.name}}
-    visualize: "true" 
-    hasHealthcheck: "{{ .Values.service.hasHealthcheck }}" 
+    visualize: "true"
+    hasHealthcheck: "{{ .Values.service.hasHealthcheck }}"
+    hasOpenAPI: "{{ .Values.service.hasOpenAPI }}" 

```

Comment:
There is none - this enables my 10% project to locate services that have a swagger `/__api` endpoint